test/test_utils.h

```diff
@@ -39,18 +39,18 @@ struct Resolution {
 
     inline bool operator <=(const Resolution& other) const
     {
-        return (width <= other.width) and (height <= other.height);
+        return (width <= other.width) && (height <= other.height);
```

Comment:
Nit: I would keep these and/or -> && and || as separate commit. same for the gnu initializers further down